## 6- A frequência de "TV Show" de todos os dados e comparativamente em relação as duas plataformas

In [8]:
# importamos a biblioteca pandas
import pandas as pd

### Considerando TODOS os dados

In [9]:
# carregamos dados do Netflix e Amazon
netflix = pd.read_csv('netflix_titles.csv')
amazon_prime = pd.read_csv('amazon_prime_titles.csv')
# concatenamos os dados
all_data = pd.concat([netflix, amazon_prime], ignore_index=True)
# filtramos apenas por filmes
all_data = all_data[all_data['type']=='TV Show']
# resetamos os indices
all_data.reset_index(inplace=True, drop=True)
# calculamos o numero de registros (frequencia) do tipo TV Show
len(all_data)

4530

### Dados do Netflix

In [10]:
# carregamos dados do Netflix
netflix = pd.read_csv('netflix_titles.csv')
# filtramos apenas por TV Show
netflix = netflix[netflix['type']=='TV Show']
# resetamos os indices
netflix.reset_index(inplace=True, drop=True)
# calculamos o numero de registros (frequencia) do tipo TV Show
len(netflix)

2676

### Dados da Amazon

In [12]:
# carregamos dados do Netflix
amazon = pd.read_csv('amazon_prime_titles.csv')
# filtramos apenas por TV Show
amazon = amazon[amazon['type']=='TV Show']
# resetamos os indices
amazon.reset_index(inplace=True, drop=True)
# calculamos o numero de registros (frequencia) do tipo TV Show
len(amazon)

1854

## Análise Comparativa
#### Na plataforma Netflix tem-se 2676 registros o que representa  2676/4530 % ou 59% do total de registros do tipo TV Show.
#### Na plataforma Amazon 1854 registros, equivalente a 1854/4530 % ou  41% do total de registros do tipo TV Show.
#### Existe uma diferença de 18% entre plataformas. A plataforma Netflix possui a maior porcentagem de registros do tipo TV Shows